keadm/cmd/keadm/app/cmd/cloud/init.go

```diff
@@ -148,7 +150,7 @@ func AddInit2ToolsList(toolList map[string]types.ToolsInstaller, initOpts *types
 		AdvertiseAddress: initOpts.AdvertiseAddress,
 		KubeEdgeVersion:  initOpts.KubeEdgeVersion,
 		Manifests:        initOpts.Manifests,
-		Namespace:        constants.SystemNamespace,
+		Namespace:        initOpts.Namespace,
```

Comment:
There are many places where  `constants.SystemNamespace` is used, It's difficult to eliminate these uses

Reply:
But I feel that `namespace` can be better selected by users themselves

Reply:
We can gradually optimize this starting from the installation. What do you think？